source/install/desktop-app-install.rst

```diff
@@ -18,159 +18,189 @@ Desktop application install guides
 
-    Unofficial, community-driven ``.deb`` packages are available.
+    - 64-bit systems:
+    
+      .. code-block:: none
 
-    1. Download the latest version of the Mattermost Desktop App:
+        sudo rpm -u mattermost-desktop-5.0.4-linux-x86_64.rpm
 
-      - 64-bit systems `mattermost-desktop-5.0.4-linux-amd64.deb <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-amd64.deb>`__
-      - 32-bit systems `mattermost-desktop-5.0.4-linux-i386.deb <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-i386.deb>`__
+    - 32-bit systems:
+    
+      .. code-block:: none
+ 
+        sudo rpm -u mattermost-desktop-5.0.4-linux-i686.rpm
 
-    2. At the command line, execute one of the following commands depending on the package that you downloaded:
+    .. tip:: 
+      You can review the current version of your Desktop App by selecting the **More** |more-icon-vertical| icon located in the top left corner of the Mattermost window, then selecting **Help > Version...**.
 
-      - 64-bit systems ``sudo dpkg -i mattermost-desktop-5.0.4-linux-amd64.deb``
-      - 32-bit systems ``sudo dpkg -i mattermost-desktop-5.0.4-linux-i386.deb``
+  .. tab:: Generic Linux
 
-    3. Run Mattermost as a desktop app.
+    A beta AppImage distribution of a compressed tarball is available. Automatic app updates aren’t supported. You must update your app manually. 
 
+    1. Download the latest version of the Mattermost Desktop App:
 
-  .. tab:: Install from Mattermost Repository
+      - 64-bit systems: `mattermost-desktop-5.0.4-linux-x64.tar.gz <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-x64.tar.gz>`__
+      - 32-bit systems: `mattermost-desktop-5.0.4-linux-ia32.tar.gz <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-ia32.tar.gz>`__
 
-    Community-driven ``.deb`` packages are published into Mattermost Repository.
-    
-    1. In a terminal window, run the following command to configure the Mattermost Repository.
+    2. Extract the archive to a convenient location, then execute ``mattermost-desktop`` located inside the extracted directory.
 
-      ``curl -o- https://deb.packages.mattermost.com/setup-repo.sh | sudo bash``
+    3. To create a Desktop launcher, open the file ``README.md``, and follow the instructions in the **Desktop launcher** section.
 
-    2. Install the Mattermost Desktop package.
+  .. tab:: macOS
 
-      ``sudo apt install mattermost-desktop``
+    MacOS 10.15+ is required. You have two ways to install the Desktop App, and how you install the app determines whether it updates automatically.
 
-    3. Run Mattermost as a desktop app.
+    **Install from the App Store**
+
+    We recommend that you install the Desktop App from the `App Store <https://apps.apple.com/us/app/mattermost/id1257222717>`__. When you install through the App Store, your Desktop App updates automatically when a new release is available.
```

Comment:
This appears to be a link to the iOS app in the app store, not the macOS app. Let me know if I'm missing something, but I only see boards as a macOS app on the app store (https://apps.apple.com/us/developer/mattermost-inc/id654282837). Did a search for Mattermost on the App Store macOS app on my Mac and only Focalboard came up.

Reply:
Great catch, @rmonga71! This link takes users to the iOS mobile app, not the Desktop App. I'll replace this link as soon as its available. Thank you!!